README.md

```diff
@@ -1,5 +1,25 @@
 # Steamy - Steampunk hero platform shooter
+
+
+### Most important things about the project:
+
+- **MVVM** - We're creating our own MVVM implementation, where models are clear classes,
+and views and view-models are monobehaviours.
+
+- **Teamwork** - This is our first big project with more than 4 members. We're learning not
+only how to work with git, but also about connecting programmers, graphic artists and designers work.
+
+- **Input system** - We created input abstraction, because Unity default implementation
```

Comment:
Nie no nie mamy abstrakcji dla inputu :/ bo brakuje nam odpowiedniego dependency injection żeby to miało szansę działać